torch/quantization/fx/quantize.py

```diff
@@ -320,22 +326,32 @@ def _prepare(self, model, qconfig_dict, inplace, is_dynamic_quant):
         self._generate_qconfig_map(model, model.graph, qconfig_dict)
 
         # match the patterns that will get quantized
-        matches = self._find_matches(model.graph, self.modules, self.patterns)
+        standalone_module_names = qconfig_dict.get('standalone_module_name', None)
+        matches = self._find_matches(
+            model.graph, self.modules, self.patterns, standalone_module_names)
 
         # find _inputs_ to matched nodes that are not quantized, these
         # have to be quantized, which requires measuring stats,
         # initialize an DefaultQuant object for each
         quants = self._find_quants(model.graph, matches)
 
         self.activation_post_process_map = dict()
-
         env = {}
         observed_graph = Graph()
         observed_node_names_set = set()
 
         def load_arg(a):
             return map_arg(a, lambda node: env[node.name])
 
+        # indexes for the inputs that needs to be observed
+        observed_input_idxs = []
```

Comment:
would it make sense to either name this `standalone_module_observed_input_idxs`, or to make it generic and also populate it for non standalone modules?

Reply:
Sure, will rename to `standalone_module_observed_input_idxs`